static/src/javascripts/projects/common/modules/analytics/google.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+
+const trackerName = config.googleAnalytics.trackers.editorial;
+const send = `${trackerName}.send`;
+
+const extractLinkText = (el: ?HTMLElement): ?string => {
```

Comment:
I don't get why the `el` is nullable. If the caller would pass null to the function, perhaps it should just not call the function? It's the caller's responsibility to not pass garbage